### PR TITLE
Added darkmode

### PR DESCRIPTION
**Pull request by TomTomRixRix**
Fixes issue #96 
Please try on your setup again, as I'm not sure if it works in the general case

### CODE REVIEW

File: frontend/src/components/Application.vue [hunk 1]
```diff
@@ -122,6 +137,23 @@ export default {
...
     }
+
+    // Dark theme if set in user preferences
+    if (
+      window.matchMedia &&
+      window.matchMedia("(prefers-color-scheme: dark)").matches
+    ) {
+      // dark mode
+      this.$vuetify.theme.dark = true;
+      alert("Dark");
```
Comment (christian-heusel) [inline frontend/src/components/Application.vue:148]: pls remove this

**Review by christian-heusel — CHANGES_REQUESTED** [review]

**Comment by christian-heusel** [comment]
For me at least this implementation does not do the initial detection right ... Does it work for you @TomTomRixRix?

File: frontend/src/components/Application.vue (hunk 1 above)
Comment (TomTomRixRix) [reply frontend/src/components/Application.vue:148]: Upsi, that's still debuggin stuff...

**Comment by TomTomRixRix** [comment]
> For me at least this implementation does not do the initial detection right ... Does it work for you @TomTomRixRix?

For me it doesn't work as well, but I'm not sure whether it's due to my setup. The same code has already worked in the [Feedbackbuch](https://github.com/FachschaftMathPhysInfo/Feedbackbuch).

**Comment by christian-heusel** [comment]
> The same code has already worked in the [Feedbackbuch](https://github.com/FachschaftMathPhysInfo/Feedbackbuch).

Strangely there it works for me 🤔 

I think I found the bug tho by comparing the sources:
```diff
-mounted: () => {}
+mounted() {}
```

**Review by christian-heusel — APPROVED** [review]

**Pull request merged**